skimage/_shared/fft.py

```diff
@@ -0,0 +1,20 @@
+"""Prefer FFTs via the new scipy.fft module when available (SciPy 1.4+)
+
+Otherwise fall back to numpy.fft.
+"""
+try:
+    import scipy.fft
+    from scipy.fft import next_fast_len
```

Comment:
Should we prefer pocketfft from numpy instead?

https://docs.scipy.org/doc/numpy/release.html

Reply:
No, because `scipy.fft` is also now using pocketfft, but a newer C++/pybind11 version called pypocketfft. One difference with NumPy's is that the scipy one does transforms in either `float`, `double` or `long double` precision while NumPy's pocketfft is `double` only.

see some details on the new `fft` module in SciPy here.
https://github.com/scipy/scipy/wiki/Release-note-entries-for-SciPy-1.4.0#scipyfft-added

Reply:
ok. well this is really confusing (but totally unreated to this PR). I wish they could have just harmonized the two.

Reply:
Well, `scipy.fft` should now be harmonized with `numpy.fft` aside from differences in dtype support and the fact that `scipy.fft` offers an additional, optional "overwrite_x" argument. Unfortunately we have to still keep `scipy.fftpack` around indefinitely for all the legacy code that is using it. The FFTPACK Fortran code has actually been removed in 1.4 (the fftpack API is still there, but is implemented via pypocketfft).

Reply:
>ok. well this is really confusing (but totally unreated to this PR). I wish they could have just harmonized the two.

NumPy doesn't accept C++ code so that's not really possible.

Reply:
Ah I see. That's really too bad